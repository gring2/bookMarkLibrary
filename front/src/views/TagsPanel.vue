<template>
  <div>
    <aside class="side">
      <ul v-for="tag in tags">
        <Tag :tag="tag"></Tag>
      </ul>
    </aside>

  </div>
</template>

<script lang="ts">
import { Component , Vue } from 'vue-property-decorator'
import Tag from '@/components/Tag.vue'
@Component({
  components: {
    Tag,
  },
})
export default class TagsPanel extends Vue {
  private tags?: string[] = []
  private created() {
    this.tags = ['test1', 'test2']
  }
}
</script>

<style scoped lang="scss">
  aside{
    width: 25%
  }
</style>