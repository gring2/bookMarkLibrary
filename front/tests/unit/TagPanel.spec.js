import { mount } from '@vue/test-utils';
import TagsPanel from '@/views/TagsPanel.vue';
describe('TagPenel Test', () => {
    it('render TagPenel view', () => {
        const wrapper = mount(TagsPanel, {
            data: () => {
                return {
                    tags: ['test tag1', 'test tag2']
                };
            }
        });
        expect(wrapper.text().indexOf('test tag1')).toBeTruthy();
        expect(wrapper.text().indexOf('test tag2')).toBeTruthy();
        expect(wrapper).toMatchSnapshot();
    });
});
//# sourceMappingURL=TagPanel.spec.js.map