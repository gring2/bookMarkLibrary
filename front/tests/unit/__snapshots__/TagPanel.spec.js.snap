// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagPenel Test render TagPenel view 1`] = `
<aside class="side">
  <ul><span>test1</span></ul>
  <ul><span>test2</span></ul>
</aside>
`;
